packages/upload-media/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://json.schemastore.org/tsconfig.json",
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types",
+		"types": [ "gutenberg-env" ]
+	},
+	"include": [ "src/**/*" ],
+	"references": [
+	  { "path": "../api-fetch" },
+	  { "path": "../blob" },
+	  { "path": "../compose" },
+	  { "path": "../data" },
+	  { "path": "../element" },
+	  { "path": "../i18n" },
+	  { "path": "../private-apis" },
+	  { "path": "../url" }
```

Comment:
Looks like the indentation with spaces on these lines (they should use tabs) is causing problems for other PRs because the pre-commit hook auto-fixes them so that it results in an unrelated changed file in other PRs. See for example https://github.com/WordPress/gutenberg/pull/67433#discussion_r1889839918
I'll try a quick PR if @swissspidy doesn't beat me to it.